src/components/Wallet/RemarkFormUser/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { TextInput } from '../../InputComponents';
+import styles from '../../Modals/styles.module.scss';
+import { encodeRemark } from '../../../api/nodeRpcCall';
+
+export default function RemarkFormUser({
+  register,
+  indexItem,
+  errors,
+  watch,
+  setValue,
+}) {
+  const idName = indexItem !== null ? `id${indexItem}` : 'id';
+  const descriptionName = indexItem !== null ? `description${indexItem}` : 'description';
+  const combined = indexItem !== null ? `combined${indexItem}` : 'combined';
+
+  const id = watch(idName);
+  const description = watch(descriptionName);
+
+  useEffect(() => {
+    const remark = {
+      id,
+      description,
+    };
+
+    encodeRemark(remark).then((encoded) => setValue(combined, encoded, { shouldValidate: true }));
+  }, [id, description, setValue, combined]);
+
+  return (
+    <>
+      <div className={styles.title}>ID</div>
+      <TextInput
+        register={register}
+        name={idName}
+        errorTitle="ID"
+        placeholder="Enter ID"
```

Comment:
ID of what?

Reply:
"For normal accounts, remark fields will be ID and Description"
we should specify this id, if yes, what should it be?